i18n/es.po

```diff
@@ -320,6 +323,9 @@ msgstr ""
 "No se puede importar: Las versiones no coinciden (base de "
 "datos={{dbVersion}}, fichero={{sheetVersion}})"
 
+msgid "User with no permissions"
+msgstr ""
```

Comment:
Could you translate all new literals to Spanish? I am aware we have already some translations missing but, as a rule-of-thumb, we try to add the new ones.